src/jdk.jlink/share/classes/jdk/tools/jlink/internal/JlinkTask.java

```diff
@@ -275,8 +275,12 @@ int run(String[] args) {
             }
 
             return EXIT_OK;
+        } catch (FindException e) {
+            log.println(taskHelper.getMessage("err.prefix") + " " + e.getMessage());
+            e.printStackTrace(log);
```

Comment:
I think my preference would be to not print the stack trace here, instead print the exception and the cause. This is to avoid users of the tool thinking that the tool "has crashed".